src/wr_entry_point/launch/test/mock_arm.launch

```diff
@@ -2,7 +2,7 @@
 <launch>
   <include file="$(find wr_entry_point)/launch/comp/comp_init.launch" />
 
-  <include file="$(find wr_hsi_roboclaw)/launch/roboclaw_real.launch">
+  <include file="$(find wr_hsi_roboclaw)/launch/roboclaw_mock.launch">
```

Comment:
Maybe condition this on `WROVER_HW` to allow for easy testing on simulation/hardware.